shared/ui-components/src/page_components/unit_tables.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react"
+import { BasicTable } from "@dahlia/ui-components/src/tables/basic_table"
+
+const unitTypeLabel = unitType => {
+  const unitTypeLabels = {
```

Comment:
Yeah I'm not keen on this approach either, but I wasn't sure where we actually do want this logic. Thanks for pointing out this could very well be something we push to the backend.